source/templates/handlebars-basics.md

```diff
@@ -11,9 +11,19 @@ write any additional code to make sure it keeps up-to-date.
 
 ### Defining Templates
 
-By default, adjust your [application template](../the-application-template), that is created automatically for you and will be displayed on the page when your app loads.
+The first thing you should change is your [application template](../the-application-template) that is created
```

Comment:
The default `application.hbs` template is:

```html
<h2 id="title">Welcome to Ember.js</h2>

{{outlet}}
```

I'd guess that most people would want to change the `<h2>` there :)